agent/bootstrap.go

```diff
@@ -844,7 +844,7 @@ func (b *Bootstrap) Start() error {
 	}
 
 	// Switch the internal wd to it
-	commentf("Switching working directroy to build folder")
+	commentf("Switching working directory build directory")
```

Comment:
Maybe it'd be better if we put it here? Since it's a header, it'd be visible each build, and for the most part that information isn't relevant when scanning down the headers.

```
commentf("Switching working directory to: %s", b.env.Get("BUILDKITE_BUILD_CHECKOUT_PATH"))
```